bin/logs

```diff
@@ -4,9 +4,4 @@ set -e
 
 SCRIPT_DIR="$(dirname "$0")"
 LOGS_DIR="$(cd $SCRIPT_DIR/../logs && pwd -P)"
-
-ls -1 -t $LOGS_DIR \
-    | grep -F -v '(?!.*(idx|siz))' \
-    | grep -v slave \
-    | head -1 \
-    | xargs -I % tail -n 500 "$@" "$LOGS_DIR/%"
+tail -f ${LOGS_DIR}/*.log
```

Comment:
I think this one should only follow the `info` logs? and `./bin/debug-logs` follows the `debug` logs?

I haven't used the tools myself, but I thin that was their original intent